src/components/templates/Auditorium/components/AllSectionPreviews/AllSectionPreviews.tsx

```diff
@@ -39,13 +43,16 @@ export const AllSectionPreviews: React.FC<SectionPreviewsProps> = ({
 
   const {
     auditoriumSections,
+    isAuditoriumSectionsLoaded,
     toggleFullAuditoriums,
     isFullAuditoriumsHidden,
     enterSection,
     availableSections,
   } = useAllAuditoriumSections(venue);
 
-  const sectionsCount = auditoriumSections.length;
+  const sectionsCount = venue.sectionsCount ?? 0;
```

Comment:
Why do we have to create another source of truth here? We already fetch all sections in `useAllAuditoriumSections` so why the need in a separate variable here?

Reply:
Really good question. So, as you know as per https://docs.google.com/document/d/1UvFuVgE7yzJSLNSTsv86LFkkd3qJ8i3CGKupfhKv5_c/edit plan, our next task is to implement `infinite scrolling`. Given that we won't fetch `allAuditoriumSections`, we won't have access to its length. Thus this variable.

However, I think it should be moved to another place(new counters collection) given that it can be updated often & it'll affect all venues in this case. which shouldn't happen